paywall/src/__tests__/data-iframe/cache/localStorage.test.js

```diff
@@ -239,10 +227,13 @@ describe('localStorage cache', () => {
       expect.assertions(1)
 
```

Comment:
the diff below is the best example of how the behavior of the `localStorage` driver has changed:

```diff
-        'unlock-protocol/123/bar': '{"fooe":"bare","bare":"bare"}',
-        'unlock-protocol/123/hi': '{"foo":"bar","bar":"bar"}',
-        'unlock-protocol/456/bar': '{"fooe":"bare"}',
-        'unlock-protocol/456/hi': '{"foo":"bar"}',
+        'unlock-protocol/123/bar#fooe': '"bare"',
+        'unlock-protocol/123/bar#bare': '"bare"',
+        'unlock-protocol/123/hi#foo': '"bar"',
+        'unlock-protocol/123/bar#foo': '"bar"',
+        'unlock-protocol/123/hi#bar': '"bar"',
+        'unlock-protocol/456/bar#fooe': '"bare"',
+        'unlock-protocol/456/hi#foo': '"bar"',
```